doc/ruleSets/index.rst

```diff
@@ -2,12 +2,12 @@
 List of Available Rule sets
 ===========================
 - `@DoctrineAnnotation <./DoctrineAnnotation.rst>`_
-- `@PER <./PER.rst>`_
-- `@PER-CS1.0 <./PER-CS1.0.rst>`_
-- `@PER-CS1.0:risky <./PER-CS1.0Risky.rst>`_
+- `@PER <./PER.rst>`_ *(deprecated)*
+- `@PER-CS1.0 <./PER-CS1.0.rst>`_ *(deprecated)*
```

Comment:
Nice to mention it here


Only to ask - do we also mention it for rules in the index file ?

Reply:
Yes, it was taken directly from the fixers' doc generator, I only decided to not display `risky` here because it's already in the name (described it in the commit message).